onnxscript/ir/serde.py

```diff
@@ -509,6 +544,7 @@ def deserialize_graph(proto: onnx.GraphProto) -> _core.Graph:
     return _deserialize_graph(proto, [])
 
 
+@_capture_errors(lambda proto: proto.name)
```

Comment:
Shouldn't the lambda have the same arguments as the function below, which has an extra scoped_values?

Reply:
Good catch thanks!